src/compiler.md

```diff
@@ -86,25 +73,34 @@ It is also often useful to emit debug information, like function names, alongsid
 There are several flags that enable or disable specific WebAssembly or compiler features. By default, only the bare minimum is exposed, and fully standardized WebAssembly features will be used.
 
 ```
---importMemory        Imports the memory provided as 'env.memory'.
+--importMemory        Imports the memory from 'env.memory'.
 --noExportMemory      Does not export the memory as 'memory'.
 --initialMemory       Sets the initial memory size in pages.
 --maximumMemory       Sets the maximum memory size in pages.
 --sharedMemory        Declare memory as shared. Requires maximumMemory.
---importTable         Imports the function table provided as 'env.table'.
+--importTable         Imports the function table from 'env.table'.
 --exportTable         Exports the function table as 'table'.
+--runtime             Specifies the runtime variant to include in the program.
+
+                        default      TLSF + lightweight GC invoked externally
+                        incremental  TLSF + incremental GC (experimental)
+                        stub         Minimal runtime stub (never frees)
+                        ...          Path to a custom runtime implementation
```

Comment:
Ooooooooooooo that's super insteresting you can add a custom runtime! That's cool! :smile: